app/views/projects/confirm_delete.html.erb

```diff
@@ -0,0 +1,18 @@
+<header class="heading-with-actions">
+  <h1><%= project.name %></h1>
+</header>
+
+<div class="lead">
+  You are about to <strong>delete</strong> this project and the project content
+  items that belong to it - this will remove the project and any suggested tags
```

Comment:
I'd suggest leaving out `and the project content items that belong to it ` as I don't think project content items is a concept that we want to expose.